airframe-sql/src/main/scala/wvlet/airframe/sql/model/Expression.scala

```diff
@@ -541,6 +545,8 @@ object Expression {
       with LogSupport {
     override def name: String = "*"
 
+    override def sqlExpr: String = name
```

Comment:
qualifier (e.g., `A.*`) can be missing from the generated sql. Is it ok?

Reply:
I removed that change as it ended up not being used. I kept the existing SQL generation logic.